test/jdk/java/lang/ProcessBuilder/PipelineLeaksFD.java

```diff
@@ -37,7 +37,7 @@
 
 /*
  * @test
- * @bug 8289643
+ * @bug 8289643 9291760
```

Comment:
Hello Roger, there's a typo here in the new bug id. Should have been `8291760`.
Other than that, the rest of the change looks fine to me.